tests/phpunit/data/wpml-config.xml

```diff
@@ -8,6 +8,7 @@
 				<custom-field action="translate">custom-description</custom-field>
 				<custom-field action="ignore">date-added</custom-field>
 				<custom-field action="translate" encoding="json">a_json_meta</custom-field>
+				<custom-field action="translate" encoding="json, urlencode, base64">an_encoded_meta</custom-field>
```

Comment:
Should we add a case with `json,urlencode,base64`? (meaning no spaces, like WordPress does for lists).

Reply:
In [1160694](https://github.com/polylang/polylang/pull/1679/commits/1160694a8cc306a82ca82f2504c0c65a0d1c518f) I removed the space characters entirely: it makes no sense to test it with them since WPML doesn't support them.